### DISCUSSION TITLE
couldnt find my site_id

### DISCUSSION THREAD

**Discussion by ghost** [Art-of-WiFi/UniFi-API-client#229] [Q&A] [answered]
Hi, I was googling for an external captive portal solution and found a video on YouTube that has a link to this GitHub repository: https://github.com/splash-networks/capport/tree/master/Unifi.

I'm new to this, so I have some questions:

1- Will this connecting.php script from the repository work?:
```
<?php

session_start();

$mac = $_SESSION["id"];
$ap = $_SESSION["ap"];
$name = $_POST['name'];
$email = $_POST['email'];

require __DIR__ . '/vendor/autoload.php';

$duration = 30; //Duration of authorization in minutes
$site_id = 'site_id'; //Site ID found in URL (https://1.1.1.1:8443/manage/site/<site_ID>/devices/1/50)

$controlleruser     = 'username'; // the user name for access to the UniFi Controller
$controllerpassword = 'password'; // the password for access to the UniFi Controller
$controllerurl      = 'https://1.1.1.1:8443'; // full url to the UniFi Controller, eg. 'https://22.22.11.11:8443'
$controllerversion  = '5.10.21'; // the version of the Controller software, eg. '4.6.6' (must be at least 4.0.0)
$debug = false;

$unifi_connection = new UniFi_API\Client($controlleruser, $controllerpassword, $controllerurl, $site_id, $controllerversion);
$set_debug_mode   = $unifi_connection->set_debug($debug);
$loginresults     = $unifi_connection->login();

$auth_result = $unifi_connection->authorize_guest($mac, $duration, $up = null, $down = null, $MBytes = null, $ap);

//User will be authorized at this point; their name and email address can be saved to some database now
?>

<!doctype html>
<html lang="en">
    <head>
        <meta charset="utf-8">
        <title>WiFi Portal</title>
        <meta name="viewport" content="width=device-width, minimum-scale=1.0, maximum-scale=1.0, user-scalable=no">
		<meta http-equiv="refresh" content="5;url=https://www.google.com/" />
    </head>
    <body>
            <p>You're online! <br>
            Thanks for visiting us!</p>
    </body>
</html>
```
(I'm unable to test it yet because I can't find my site ID.)

2 - Where do I find my site ID? It should be in my URL, but I can't find it (and I have more than one network server).

Sorry if these are dumb questions, but I really can't find any answers in my searches.

**Comment by thib3113** [comment, marked as answer]
To get your site url, you can go to your dashboard, and get it from url : 

on unifiOs : `https://unifi/network/<siteID>/dashboard` ( the default site is called "default" ) . 
on unifi : `https://unifi/manage/<siteID>/dashboard`
using unifi.ui.com : `https://unifi.ui.com/network-servers/<uuidOfYourInstall>/network/<siteId>/dashboard`

**Comment by DonBower** [comment]
This worked for me:
`curl -k -X GET 'https://10.0.0.1/proxy/network/integrations/v1/sites' -H 'X-API-KEY: <API-Key-Redacted> -H 'Accept: application/json' | jq .`
